src/Sdk/Cerbos/Sdk/Cloud/V1/HubClientBuilder.cs

```diff
@@ -18,35 +18,67 @@ public sealed class HubClientBuilder
     {
         private string Target { get; }
         private Credentials Credentials { get; set; }
+        private const string DefaultTarget = "https://api.cerbos.cloud";
 
-        private HubClientBuilder(string target)
+        private HubClientBuilder(string target, string clientId, string clientSecret)
         {
             Target = target;
+            Credentials = new Credentials(clientId, clientSecret);
         }
 
-        public static HubClientBuilder ForTarget(string target)
+        public static HubClientBuilder NewInstance(string target, string clientId, string clientSecret)
```

Comment:
I don't think `target` should be a public parameter at all. There should be just two static constructors: `FromEnv` and `WithCredentials`. See https://github.com/cerbos/cerbos-sdk-java/blob/main/src/main/java/dev/cerbos/sdk/hub/CerbosHubClientBuilder.java.